src/components/player-info.js

```diff
@@ -65,6 +66,18 @@ AFRAME.registerComponent("player-info", {
 
     const modelEl = this.el.querySelector(".model");
     if (this.data.avatarSrc && modelEl) {
+      modelEl.addEventListener(
```

Comment:
Rather than doing the preloading here I would just do it in ui-root where you are early-updating player-info.